pkgs/os-specific/linux/multipath-tools/default.nix

```diff
@@ -21,20 +22,29 @@
 
 stdenv.mkDerivation rec {
   pname = "multipath-tools";
-  version = "0.9.6";
+  version = "0.9.8";
 
   src = fetchFromGitHub {
     owner = "opensvc";
     repo = "multipath-tools";
     rev = "refs/tags/${version}";
-    sha256 = "sha256-X4sAMGn4oBMY3cQkVj1dMcrDF7FgMl8SbZeUnCCOY6Q=";
+    sha256 = "sha256-4cby19BjgnmWf7klK1sBgtZnyvo7q3L1uyVPlVoS+uk=";
   };
 
+  patches = [
+    # Backport build fix for musl libc 1.2.5
+    (fetchpatch {
+      url = "https://github.com/openSUSE/multipath-tools/commit/e5004de8296cd596aeeac0a61b901e98cf7a69d2.patch";
+      hash = "sha256-ZvNFVphB9f+S/XMxktR6P/YYSTLeJXEsj4XrAnw6GUI=";
+      excludes = ["tests/util.c"];
+    })
+  ];
+
   postPatch = ''
     substituteInPlace create-config.mk \
       --replace /bin/echo ${coreutils}/bin/echo
 
-    substituteInPlace multipathd/multipathd.service \
+    substituteInPlace multipathd/multipathd.service.in \
       --replace /sbin/multipathd "$out/bin/multipathd"
```

Comment:
We should replace this with replace-fail

Reply:
replace-warn matches better the previous usage and if we would have changed that, we could have expressed hundreds of none matching lines which then would have been breakages